### PR TITLE
Remove Chart JSON Data from views, urls and templates

### PR DESCRIPTION
**Pull request by danielabertranou-eb**

### CODE REVIEW

File: revenue_app/views.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import json
```
Comment (eb-francisco) [inline revenue_app/views.py]: I think this is not in use, you can safely remove it

Comment (facundochaud-eb) [reply revenue_app/views.py]: It's used in the last two function based views

**Comment by danielabertranou-eb** [comment]
> Could you please provide a better description of what this is not needed anymore? Sorry I have not the entire context :)

We removed two views as they were just a placeholder for the dashboard template. Next time I will explained it better on the commit message.

**Comment by eb-francisco** [comment]
> > Could you please provide a better description of what this is not needed anymore? Sorry I have not the entire context :)
> 
> We removed two views as they were just a placeholder for the dashboard template. Next time I will explained it better on the commit message.

thanks!

File: revenue_app/views.py (hunk 1 above)
Comment (eb-francisco) [reply revenue_app/views.py]: nvm then, completely missed that one

**Pull request merged**